### PR TITLE
Some rewording about import predef

### PR DESCRIPTION
**Pull request by som-snytt**

### CODE REVIEW

**Comment by som-snytt** [comment]
I bet they could use this info over on the docs site.

File: puzzlers/pzzlr-046.html [hunk 1]
```diff
@@ -73,9 +73,31 @@ <h3>Explanation</h3>
...
+       new Object() + "... I object!"
+                    ^
+
+scala> import Predef._
+import Predef._
+
+scala> new Object() + "... I object!"
+res2: String = java.lang.Object@7857cb1d... I object!
+</pre>
+When pasting into the REPL, <tt>:paste -raw</tt> will preserve the behavior of the special import.
```
Comment (demobox) [inline puzzlers/pzzlr-046.html]: Do you think the `:paste -raw` bit is worth keeping? I think it was included in the original version mainly as a way to make it work.

**Comment by demobox** [comment]
Merged to [master](82d465b). Thanks, @som-snytt!

**Comment by demobox** [comment]
image

File: puzzlers/pzzlr-046.html (hunk 1 above)
Comment (som-snytt) [reply puzzlers/pzzlr-046.html]: `:pa -raw`  is ordinary compilation, so it just works. Yes, it's the other mechanism that needs advertising. Anyone who would `:pa -raw` knows how.

**Comment by som-snytt** [comment]
Thanks to you. I had forgotten how this worked in REPL (even though I added it). It doesn't work for `{import Predef.etc ; new Object() + " argh"}` though it could be made to accommodate that.

**Pull request closed without merging**